cognite/client/data_classes/geospatial.py

```diff
@@ -169,6 +169,10 @@ def _is_reserved_property(property_name: str) -> bool:
     return property_name.startswith("_") or property_name in RESERVED_PROPERTIES
 
 
+def to_feature_property_name(property_name: str) -> str:
```

Comment:
Do you want this to be part of the public API? If not, mark it as protected with a leading underscore.
```suggestion
def _to_feature_property_name(property_name: str) -> str:
```